app/decorators/io_decorator.rb

```diff
@@ -0,0 +1,9 @@
+class IoDecorator < Hydra::Derivatives::IoDecorator
+  def original_filename
```

Comment:
Does browse_everything want different methods for filename and mime type than what hydra-derivatives uses?  Seems strange that there would be a mismatch.